eng/pipelines/templates/steps/build.yml

```diff
@@ -73,8 +73,11 @@ steps:
         New-Item -Type Directory -Name $artifactName -Path $(Build.ArtifactStagingDirectory) > $null
         Copy-Item sdk/${{parameters.ServiceDirectory}}/**/$artifactName-*.tgz $(Build.ArtifactStagingDirectory)/$artifactName
         Copy-Item sdk/${{parameters.ServiceDirectory}}/**/browser/$artifactName-*.zip $(Build.ArtifactStagingDirectory)/$artifactName
-        New-Item -Type Directory -Name documentation -Path $(Build.ArtifactStagingDirectory)/$artifactName > $null
-        Copy-Item $(Build.SourcesDirectory)/docGen/$artifactName.zip $(Build.ArtifactStagingDirectory)/$artifactName/documentation
+        if (Test-Path -Path $(Build.SourcesDirectory)/docGen/$artifactName.zip)
```

Comment:
Do we have valid scenarios where we don't have a generated docs artifact? Does it output information other places in the logs somewhere of why it didn't get generated?

Reply:
yes. My change has broken test-utils pipeline. https://dev.azure.com/azure-sdk/public/_build?definitionId=940&_a=summary

docgen is skipped for private packages.

Reply:
Here is successful pipeline after fixing this issue. https://dev.azure.com/azure-sdk/public/_build/results?buildId=955285&view=logs&j=3dc8fd7e-4368-5a92-293e-d53cefc8c4b3&t=e2744c02-d5ab-5ec4-b597-2a76a072b500

Reply:
I just want to make sure we don't accidently start missing docs because something that was supposed to generate them didn't generate them and failed silently. So it might be worth making this only skip for things that are opt-ed in.

Would checking https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/test-utils/ci.yml#L28 be a reasonable way to determine that we skipped docs?

Reply:
I think this is a good suggestion. Let me check